### PR TITLE
Some minor bug fixes and some cleaning

### PR DESCRIPTION
**Pull request by schustermartin**
- Fixed some edge cases where in(...) function was returning false results due to forth and back conversion between Cylindrical and Cartesian Points. This is done by skipping trigonometric conversions where unnecessary.
- Fixed in(...) function for the Cone Primitive. So far translate vector was not taken into account.
- Switched to the use of RotationsPackage for VolumePrimitives (not used so far, just how general structure could look like).
- Got rid of some unnecessary lines of code (expressing the same in less lines).

### CODE REVIEW

File: src/Geometries/VolumePrimitives/Tube.jl [hunk 1]
```diff
@@ -56,20 +56,14 @@ function Geometry(T::DataType, t::Val{:tube}, dict::Dict{Union{Any,String}, Any}
 end
 
 function in(point::CartesianPoint{T}, tube::Tube{T}) where T
-    ismissing(tube.translate) ? nothing : point -= tube.translate
+    (ismissing(tube.translate) || tube.translate == CartesianVector{T}(0.0,0.0,0.0)) ? nothing : point -= tube.translate
```
Comment (fhagemann) [inline src/Geometries/VolumePrimitives/Tube.jl:59]: `tube.translate` should never be zero

**Review by fhagemann — APPROVED** [review]
Right now, the `translate` vector of `Tube` should never be zero. But we can leave it like this.

File: src/Geometries/VolumePrimitives/Tube.jl (hunk 1 above)
Comment (lmh91) [reply src/Geometries/VolumePrimitives/Tube.jl:59]: Well, we have to change the general struct of our primitives anyhow when we add Rotations. 
This `if ismissing` should never appear. I guess we will use parametric types to handle this on compilation level.

Comment (lmh91) [reply src/Geometries/VolumePrimitives/Tube.jl:59]: For both, translation and rotation.

**Review by lmh91 — APPROVED** [review]

**Pull request merged**